docs/release-notes/feature-node-selectors.rst

```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+**New Features**
+
+-  Kubernetes Configuration: Allow cluster administrators to define Determined resource pools on
+   Kubernetes in terms of node selectors and/or affinities. Clusters administrators configure
+   resource pool's node selectors and/or affinities on the default pod spec level, under the
+   task_container_defaults.cpu_pod_spec or task_container_defaults.gpu_pod_specs. In this way,
+   cluster administrators can split a single cluster into several resource pools by way of node
+   labels.
```

Comment:
- Kubernetes Configuration: Allow Cluster administrators to define Determined resource pools on Kubernetes using node selectors and/or affinities. Configure these settings at the default pod spec level under ``task_container_defaults.cpu_pod_spec`` or ``task_container_defaults.gpu_pod_spec``. This allows a single cluster to be divided into multiple resource pools using node labels.